README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.5+](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@Zangetsu38 It's about stating the smallest version number that you need to compile. 2.7+ is fine - states that 2.7 and any versions above work. I still see no reason to change this, as **any** version above 2.7 will work (that of course includes the 3.* releases. There's a difference between, what's minimum requirement and what's currently recommended.